Commiting Test Folders (#62)
hentt30/minushalf: tests/README.txt

Reviewer: If there is no reason against it, I would recommend having this README file in markdown format

VASP Test Setup Instructions

> To run the VASP tests, you must provide the required POTCAR files for the following elements:

Reviewer: ```suggestion
To run the VASP tests, the required POTCAR files for the following elements must be provided:
```

Ag
C
F
S
Sb

> Place these files in the directory:

Reviewer: ```suggestion
Place these files in the following directory:
```

> test_farm/VESP/POTFILES

Reviewer: ```suggestion
test_farm/VASP/POTFILES
```

> Each file must be named using the following convention:

Reviewer: ```suggestion
Each file must follow this naming scheme:
```

POTCAR.<element>

where <element> is written in lowercase. For example:

POTCAR.ag
POTCAR.c
POTCAR.f
POTCAR.s
POTCAR.sb

> Ensure that all files are correctly named and located before running the tests.

Reviewer: In my opinion, this line seems redundant and could be removed
